modules/sh-camera/modules/sh-camera/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { useRef } from 'react';
+import { Text, View, TouchableOpacity } from 'react-native';
+import { styles } from './styles';
+import ActionSheet from 'react-native-actionsheet'
+import { pickFromCamera, pickFromGallery, uploadImage} from './cameraUtils'
+
+
+const shCamera = ({ navigation }) => {
+  // More info on all the options is below in the API Reference... just some common use cases shown here
+  const actionSheet = useRef(null);
+
+  const ImagePickerOptions = ['Take Photo', 'Choose from Gallery', 'Cancel'];
+
+
+  return (
+    <View
+      style={{
+        flex: 1,
+      }}>
+      <View style={styles.heading}>
+        <TouchableOpacity
+          style={styles.touchableopacity}
+          onPress={() => {
+            navigation.goBack();
+          }}>
+        </TouchableOpacity>
+        <Text style={styles.header}>Camera Module</Text>
+
+      </View>
+      <ActionSheet
+        useNativeDriver={false}
```

Comment:
I don't see this prop in their documentation, can you guide me with where this can be found and what it does?

Reply:
yeah, I could not find it either, don't recall why I added it maybe during some copy-paste it's left there. Good catch thank you.